portal/src/main/webapp/js/lib/igv.js

```diff
@@ -5122,10 +5122,41 @@ var igv = (function (igv) {
             }
             return;
         }
+        var div = $("#igvDiv")[0],
+            options = {
+                showNavigation: true,
+                showRuler: true,
+                genome: "hg19",
+                locus: feature,
+                tracks: [
+                    {
+                        url: "http://localhost:8080/cbioportal/api/copynumbersegments",
```

Comment:
Should localhost... be here?

Ideally, igv.js should not worry about the sementics of the URL. Maybe a call back function should be pass in to igv instead of url?